src/tree/render.js

```diff
@@ -23,7 +23,12 @@ const filterTree = (id, tree) => {
       currentNode.data.id === node.data.id ||
       (currentNode.parent && node.data.id === currentNode.parent.data.id) ||
       (currentNode.parent && node.parent && node.parent.data.id === currentNode.parent.data.id) ||
-      (node.parent && node.parent.data.id === currentNode.data.id)
+      (node.parent && node.parent.data.id === currentNode.data.id) ||
+      (currentNode.parent &&
```

Comment:
Show the current node's sibling's children? Nieces I guess 😅

Reply:
yea, this part is not working properly anyway. Basically I wanted to show all subtrees without children as vertical stacks, but it breaks down for un-even trees
![image](https://user-images.githubusercontent.com/6318307/72595674-a772db80-390a-11ea-952c-da3135534e68.png)